dencam/gui.py

```diff
@@ -103,8 +103,11 @@ def _update(self):
 
         self.recorder.update_timestamp()
 
-        if (self.state.value >= self.STATE_LIST.index('NetworkPage')
-              and self.state.value <= self.STATE_LIST.index('BlankPage')):
+        networkp_index = self.STATE_LIST.index('NetworkPage')
```

Comment:
is the indentation here correct? also, these index variables dont need to exist, you can directly compare self.state.value  to  self.STATE_LIST.index('BlankPage')